### ISSUE TITLE
self_knowledge() faulty

### ISSUE THREAD

**Issue by angband-import** [angband/angband#759]
**Reported by takkaria on 9 Jul 2007 02:15 UTC**
Contains:
```
if (f2 & TR2_RES_DARK)
{
	info[i++] = "You are resistant to darkness.";
}
if (f2 & TR2_RES_DARK)
{
	info[i++] = "Your eyes are resistant to blindness.";
}
```

altered in [9a5b05e] (SVN r250).

**Comment by angband-import** [comment]
**Modified by ajps on 13 Jul 2007 14:26 UTC**

**Comment by angband-import** [comment]
**Comment by ajps on 13 Jul 2007 14:33 UTC**
Fixed in [329] merged in [5dfe994] (SVN r399).
